### PR TITLE
Styles and pop up positioning

### PR DESCRIPTION
**Pull request by sheela-svmx**
- The pop up was offset on the custom marker.
- The custom marker being offset from the route end

Reference Jira

https://servicemax.atlassian.net/browse/UI-2467

screen shot 2018-09-17 at 12 54 19 pm

### CODE REVIEW

File: sass/_map-icon-custom.scss [hunk 1]
```diff
@@ -23,37 +23,37 @@
...
 
   &.content-icon {
-    width:12px;
-    height:12px;
-    margin-left: -11px; 
-    margin-top: -11.5px
+    width: 12px;
+    height: 12px;
+    margin-left: 5px; 
+    margin-top: 4.5px
```
[the remark this answers is not in the document]
Comment (sheela-svmx) [reply sass/_map-icon-custom.scss]: BUt they make a difference right? visually, that 0.5 px?  that home icon needed to be 0.5 px up

**Comment by shao** [comment]
Are we ready to merge to try things out?

**Comment by sheela-svmx** [comment]
@shao  yes I was creating a related PR for predix side so we can test out https://github.com/ServiceMax-Engineering/ui-components-predix/pull/219

**Comment by sheela-svmx** [comment]
2 approvals merging this and will publish the tag

**Pull request merged**